dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java

```diff
@@ -17,7 +17,7 @@
 package com.google.cloud.tools.opensource.classpath;
 
 import static com.google.common.base.Preconditions.checkNotNull;
-
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
Add an empty line between static import and non-static import.

Reply:
done